{"status":"FAIL","msg":"FAIL:\n Uncaught: { message: \"The Object constructor has a 'length' own property\", name: \"Test262Error\" }\n    at ../../test262/harness/assert.js:18\n    at assert (../../test262/test/built-ins/Object/S15.2.3_A3.js:9)\n\n","path":"../../test262/test/built-ins/Object/S15.2.3_A3.js","memory_kb":0,"duration":1938576641}